sdk/documentintelligence/Azure.AI.DocumentIntelligence/src/OperationDetails.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.AI.DocumentIntelligence
+{
+    public partial class OperationDetails
+    {
+        /// <summary> Operation ID. </summary>
+        public string OperationId { get; }
```

Comment:
Forcibly making this a `string` here since the service incorrectly defined it as a `Guid`.